usaspending_api/common/api_request_utils.py

```diff
@@ -212,6 +212,11 @@ def create_q_from_filter(self, filt):
                 else:
                     # Otherwise, use built in django in
                     operation = "__in"
+            if operation is '__icontains' and isinstance(value, list):
```

Comment:
The Django implementation of ArrayField does not override "icontains", it only works using the "contains" method; and due to that is always case sensitive. I can add some extra tests.